### PR TITLE
Startup Wizard

### PR DESCRIPTION
**Pull request by aryamanvinchhi**

### CODE REVIEW

File: mscore/musescore.cpp [hunk 1]
```diff
@@ -6180,6 +6167,22 @@ int main(int argc, char* av[])
...
+                  StartupWizard* sw = new StartupWizard;
+                  sw->exec();
+                  s.setValue("firstStart", false);
+                  s.setValue("keyboardLayout", sw->keyboardLayout());
+                  s.setValue("language", sw->language());
+                  delete sw;
+            }
+            QString keyboardLayout = s.value("keyboardLayout").toString();
+            StartupWizard::autoSelectShortcuts(keyboardLayout);
+            setMscoreLocale(s.value("language").toString());
```
Comment (aryamanvinchhi) [inline mscore/musescore.cpp]: @lasconic I use setMscoreLocale() here but when it loads nothing seems to change. Do you know why?

File: mscore/startupWizard.h [hunk 2]
```diff
@@ -34,15 +34,28 @@ class StartupWizardPage1 : public QWizardPage {
...
 
+class StartupWizardPage2 : public QWizardPage {
+      Q_OBJECT
+
+      QComboBox* _languages;
+
+      public:
+        StartupWizardPage2(QWidget* parent = 0);
+        QString language()    {return _languages->currentText();}
+        void init();
```
Comment (Jojo-Schmitz) [inline mscore/startupWizard.h]: 2 space indent??

File: mscore/startupWizard.cpp [hunk 3]
```diff
@@ -84,6 +84,24 @@ StartupWizardPage1::StartupWizardPage1(QWidget* parent)
       setLayout(layout);
       }
 
+StartupWizardPage2::StartupWizardPage2(QWidget* parent)
```
Comment (Jojo-Schmitz) [inline mscore/startupWizard.cpp]: Do we really need a new page, can't it get combined with the existing one, which isn't exactly full?

**Comment by aryamanvinchhi** [comment]
@Jojo-Schmitz @lasconic I have added two more pages for selecting language and workspace. If we want to move language to the first page, I can make those changes. If not, I think these two are also ready to use.

**Comment by lasconic** [comment]
Your commit history for this PR is not right. One of my commit in there. https://github.com/musescore/MuseScore/pull/3241/commits

**Comment by aryamanvinchhi** [comment]
Going to switch to a new, clean branch.

**Pull request closed without merging**